api/.dockerignore

```diff
@@ -1,4 +1,5 @@
 /target
-/Dockerfile*
+/tests
```

Comment:
With this change no tests are run in the CI anymore. Is it really what you want?

Reply:
Added Rust CI in https://github.com/swissgeol/ngm/pull/827/commits/64d2e491ea8b53a42bd1f8ab13bee9b45c89a357